sdk/sdk/pom.xml

```diff
@@ -48,6 +48,28 @@
       <artifactId>httpclient</artifactId>
       <version>${httpclient.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.parquet</groupId>
+      <artifactId>parquet-avro</artifactId>
+      <version>1.11.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-exec</artifactId>
+      <version>3.1.0</version>
```

Comment:
which code need this hive-exec dependency ?

Reply:
because you need 3.1.0 I think you are excluding this hive-exec on other pom which is bringing different version.

So, I want to know who needs this exact hive-exec and why cant they use current hive-exec

Reply:
ORCReader related API depends on this hive-exec dependency.
From other poms, I need to exclude this because somehow it was making transitive circular dependency.
We can use other hive versions also but I thought 3.1.0 is the latest so I had put this.

Reply:
please try to add dependency of other version which is already coming and try it out. Adding new version has caused multiple version jars in class path